AppKit/CPView.j

```diff
@@ -796,7 +796,7 @@ var CPViewFlags                     = { },
 
     do
     {
-        if (view == aView)
+        if (view === aView)
             return YES;
     } while(view = [view superview])
```

Comment:
@aparajita i did not find the newline explicitly demanded here: https://github.com/cappuccino/cappuccino/blob/master/CONTRIBUTING.md#cappuccino-coding-style-guidelines